Tests/StreamChatTests/StreamChatTestPlan.xctestplan

```diff
@@ -1,32 +1,37 @@
 {
-  "configurations" : [
+  "configurations": [
     {
-      "id" : "5358EB81-7D4D-42F7-B281-016C02706988",
-      "name" : "Configuration 1",
-      "options" : {
-
+      "id": "5358EB81-7D4D-42F7-B281-016C02706988",
+      "name": "Configuration 1",
+      "options": {
       }
     }
   ],
```

Comment:
What is the purpose of changing this file?

Reply:
I can see it is just format changes, probably it was done by the pre-commit hook? And because I messed with the testplan files 🤔 I can revert this if necessary